.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Bug report
+about: Create a report to help us reproduce and correct the bug
+title: ''
+labels: Bug
+assignees: ''
+
+---
```

Comment:
add something e.g. Before submitting a bug, please make sure the issue hasn't been already addressed by searching through the past issues.

Reply:
Oh yes thanks remind me